server/game/cards/03-WC/BlasterCard.js

```diff
@@ -0,0 +1,47 @@
+const Card = require('../../Card.js');
+
+class BlasterCard extends Card {
+    getAttachedAbility(ability, blasterName, officerName) {
+        let choices = {
+            'Deal 2 damages': () => true
+        };
+
+        choices[`Move ${blasterName}`] = () => true;
```

Comment:
Why not include this in the above initialiser?

Reply:
I don't think we can have an expression as a json key. I get an error. That's why I used brackets.